doc/_includes/tutorials/common/uploading-sample-files.md

```diff
@@ -1,11 +1,17 @@
-Sequence files can be uploaded by clicking on the "Upload Sequence Files" button, on the left hand side side of the sequence file table.  Files must have the extension `.fastq` or `.fastq.gz`, all other formats will be ignored.
+### Sequence Files
+
+Sequence files can be uploaded by hovering over the "Upload Files" above the sequence file table, and select the "Upload Sequence Files" drop down option. Files must have the extension `.fastq` or `.fastq.gz`, all other formats will be ignored.
 
 ![Upload sequence file.]({{ site.baseurl }}/images/tutorials/common/samples/upload-sample-file-btn.png)
 
-You can select single or multiple files in the system file selection window.
+You can select single or multiple files in the system file selection window.  If you are uploading paired end data, ensure they are selected in the same upload.
 
-![Upload File Selection.]({{ site.baseurl }}/images/tutorials/common/samples/upload-sample-file-selection.png)
+![Cancel upload]({{ site.baseurl }}/images/tutorials/common/samples/upload-sample-progress-cancel.png)
 
-Files will upload immediately and a progress bar will be displayed.  If you need to cancel an upload click the <img class="inline" src="{{ site.baseurl }}/images/tutorials/common/samples/upload-sample-cancel-btn.png" alt="Cancel Upload Button" /> button.
+Files will upload will begin immediately.  A notification will open in the bottom right corner of the screen displaying the status of the upload.  If you need to cancel an upload click the <img class="inline" src="{{ site.baseurl }}/images/tutorials/common/samples/upload-sample-cancel-btn.png" alt="Cancel Upload Button" /> button.  **Do not leave or refresh the page as this will cancel the upload**, a warning will be display if you attempt to do this.
 
-![Cancel upload]({{ site.baseurl }}/images/tutorials/common/samples/upload-sample-progress-cancel.png)
+### Assembly Files
+
+Assembly files can be uploading the same way as sequence files, using the "Upload Files" drop down above the sequence files table.  Select the "Upload Assembly" drop down option.  Assembly files must have the extension `.fasta` or `.fna`, all other formats will be ignored.
+
+![Upload assembly.]({{ site.baseurl }}/images/tutorials/common/samples/upload-assembly.png)
```

Comment:
Concatenate files image below here is still the old style.

Reply:
Updated in d7be5124f282eee310baffc0bffe627033cfcc86